app/models/collection.rb

```diff
@@ -74,7 +81,6 @@ def delete_permission(current_user, removed_user)
         errors.add(:delete_permission, "Cannot remove yourself from a collection. Contact a super-admin for help.")
       else
         errors.delete(:delete_permission)
-        removed_user.remove_role :collection_admin, self
```

Comment:
Maybe retitle `delete_permission`? I'm not sure which permission I should expect to see removed here.

Reply:
I am sorry, this was removed in error during a rebase.